#!/usr/bin/env python
PACKAGE = "trimod_gripper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("noise_axial_offset", double_t,    0, "Axial Noise Offset", 0.4,  0.0, 3.0)
gen.add("noise_axial_coefficient", double_t, 0, "Axial Noise Coefficient",    0.0019, 0.0,   1.0)
gen.add("noise_axial_bias", double_t, 0, "Axial Noise Bias",    0.0012, 0.0,   1.0)
gen.add("noise_lateral_coefficient", double_t, 0, "Lateral Noise Coefficient",    0.815, 0.0,   4.0)
gen.add("noise_lateral_focal", double_t, 0, "Lateral Noise Focal",    585, 0,   5000)


exit(gen.generate(PACKAGE, "trimod_gripper", "LwrManualPhotographer"))
